xblockutils/public/studio_edit.js

```diff
@@ -65,6 +66,11 @@ function StudioEditableXBlockMixin(runtime, element) {
                 }
             });
         }
+
+        if (type == 'datepicker' && datepickerAvailable) {
+            $field.datepicker('destroy');
+            $field.datepicker({dateFormat: "m/d/yy"});
```

Comment:
:+1: @bradenmacdonald 

(Even though I agree with xkcd, most people would prefer their localized dates, so I believe that the ticket should be about using the user’s locale to determine the date format).